src/renderer/components/Switch.js

```diff
@@ -31,14 +32,25 @@ const Base: ThemedComponent<{
 `;
 
 const Ball = styled.div`
-  width: ${p => (p.small ? 9 : 20)}px;
-  height: ${p => (p.small ? 9 : 20)}px;
+  width: ${p => (p.small ? 9 : p.medium ? 14 : 20)}px;
+  height: ${p => (p.small ? 9 : p.medium ? 14.5 : 20)}px;
   border-radius: 50%;
   background: ${p => p.theme.colors.palette.primary.contrastText};
   box-shadow: 0 1px 3px 0 rgba(0, 0, 0, 0.2);
   transition: 250ms ease-in-out transform;
   transform: translate3d(
-    ${p => (p.small ? (p.isChecked ? "14px" : "2px") : p.isChecked ? "18px" : "2px")},
+    ${p =>
+      p.small
+        ? p.isChecked
+          ? "14px"
+          : "2px"
+        : p.medium
```

Comment:
What happens here if both `small` and `medium` are passed as props ?

I think it's better to have a `size="small|medium|default|whatevs"` here (or something similar)